spec/support/fake_analytics.rb

```diff
@@ -1,4 +1,31 @@
 class FakeAnalytics
+  PiiDetected = Class.new(StandardError)
+
+  module PiiAlerter
+    def track_event(event, attributes = {})
+      raise PiiDetected, 'pii detected in analytics' if attributes.to_json.include?('pii')
+
+      pii_attr_names = Pii::Attributes.members.map(&:to_s)
```

Comment:
Would it be easier / more efficient to switch the check below to `pii_attr_names.include(value.to_sym.downcase)` to avoid the `map` here?

Reply:
I forgot that `Symbol#downcase` exists, and I have habits of avoiding `Strin#to_sym` from the old days when that was a DOS vector... will update!

Reply:
updated in c30d59c53ea7004b52fed6c304c76fedce9a9999